src/utils/tree-utils.js

```diff
@@ -181,11 +186,11 @@ export function createNodes(matrix, attributeIndecies, status, navigationMode, t
     const parentNode = nodeMap[node.parentId];
     node.parent = parentNode;
     if (parentNode) {
-      parentNode.children.length > 100
+      parentNode.children.length > 98
```

Comment:
Changed this to 98, so that the max number of children becomes 99, which fits in the +/- box.